silkworm/db/chaindb.hpp

```diff
@@ -24,11 +24,15 @@
 
 #include <lmdb/lmdb.h>
 
+#include <boost/algorithm/string.hpp>
+#include <boost/filesystem.hpp>
+#include <boost/interprocess/mapped_region.hpp>
```

Comment:
It looks like the includes are only used in the .cpp file, pls move them there.

Reply:
True for algorithm and interprocess
Filesystem is instead imported at higher level by /cmd utilities (through this .hpp)